exasol/python_extension_common/deployment/language_container_deployer.py

```diff
@@ -306,7 +310,10 @@ def create(cls,
                path_in_bucket: str = '',
                use_ssl_cert_validation: bool = True, ssl_trusted_ca: Optional[str] = None,
                ssl_client_certificate: Optional[str] = None,
-               ssl_private_key: Optional[str] = None) -> "LanguageContainerDeployer":
+               ssl_private_key: Optional[str] = None,
+               extract_timeout: timedelta = timedelta(seconds=10),
```

Comment:
A proper value for a timeout is beyond 10 minutes

Reply:
Which value do you propose?

Reply:
I will set it to 10 minutes, see next push.